include/common-types.h

```diff
@@ -0,0 +1,25 @@
+#ifndef CONNECT_SDK_COMMON_TYPES_H
+#define CONNECT_SDK_COMMON_TYPES_H
+
+#include <string>
+#include <vector>
+#include <map>
+#include <boost/move/unique_ptr.hpp>
+#include <boost/chrono/system_clocks.hpp>
+
+namespace prism {
+namespace connect {
+
+using std::string;
```

Comment:
In described scenario everything will be fine.
Please, propose a way to not say std::string each time in header, as we aren't using string other than std::string and it is unlikely we'll do it in future. And if we do, then we can just use fully qualified type e.g. boost::string for *another* string type.